e2e/blocktransfer.test.js

```diff
@@ -38,7 +39,7 @@ const {
 
 describe('Block Transfer e2e test', function() {
     this.timeout(60000);
-    async function doBlockUpload(fileUrl, fileSize) {
+    async function doBlockUpload(fileUrl, fileSize, filePath) {
```

Comment:
is `filePath` not used in this function ? its [hardcoded at line 62](https://github.com/adobe/node-httptransfer/pull/83/files#diff-7723a12188d79813c09f606e9871007fab2542abd0d97de9527756db5bf7164eR62)

Reply:
yes it is hardcoded for the test.
it is used for committing the blocks: https://github.com/adobe/node-httptransfer/blob/5405cfe7e2385acc0b87685e61265a4dc217c0ea/e2e/blocktransfer.test.js#L75